clash/bloody_grail.tex

```diff
@@ -123,8 +123,8 @@ \subsection*{\MakeUppercase{Additional Rules}}
 
 \begin{tikzpicture}[overlay]
   \centering
-  \node at (4.5, -5) {\includegraphics[scale=0.45]{\_assets/maps/bloody-grail-2p.png}};
+  \node at (4.5, -5) {\includegraphics[scale=0.165]{\_assets/maps/bloody-grail-2p.png}};
```

Comment:
This is an example where you need only 20% of the actual size of the image.

Reply:
But ghostscript reduces the file size from 73 to 15 MB :O

Reply:
So I guess we're safe with those sizes for now.

Reply:
The problem with compressing images to 20%, or any other smaller size, is that if we suddenly decide to make it larger, we need to recreate it from scratch.

On the other hand, these large images are quite heavy even on pdf browsing. I noticed slower rendering on M1 Mac.

So in the end I will probably resize them a bit but maybe to like 75% of the current size or so.